packages/code-du-travail-nlp/api/sem_search.py

```diff
@@ -0,0 +1,146 @@
+import json
+import tensorflow as tf
+import tensorflow_hub as hub
+import numpy as np
+import tf_sentencepiece
+import unidecode
+
+from typing import List
+
+content_path = "./data/content.json"
+stops_path = "./data/stops.txt"
+
+cache_path = "~/Downloads"
+
+stringvec = List[str]
+
+
+def add_slash(c):
+    c["slug"] = "/{}".format(c["slug"])
```

Comment:
en général c'est plus rapide (et plus clair je trouve) d'utiliser les string expressions
```suggestion
    c["slug"] = f"/{c['slug']}"
```